examples/tpm2-ekcert/main.go

```diff
@@ -0,0 +1,61 @@
+// Binary tpm2-ekcert reads an x509 certificate from a specific NVRAM index.
+package main
+
+import (
+	"crypto/x509"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpmutil"
+)
+
+var (
+	tpmPath = flag.String("tpm-path", "/dev/tpm0", "Path to the TPM device (character device or a Unix socket)")
+	// Default value is defined in section 7.8
+	// https://trustedcomputinggroup.org/wp-content/uploads/TCG-TPM-v2.0-Provisioning-Guidance-Published-v1r1.pdf
+	certIndex = flag.Uint("cert-index", 0x01C00002, "NVRAM index of the certificate file")
+	outPath   = flag.String("output", "", "File path for output. Leave blank to write to stdout.")
+)
+
+func main() {
+	flag.Parse()
+
+	if *certIndex == 0 {
```

Comment:
Now that a sensible default has been set, I'm not sure this check is relevant anymore. It was checking for 0 in the past because that was the default in the flag. How about just removing this?

Reply:
You're right, it's unnecessary now. Removed